src/components/AutoCompleteSuggestions/AutoCompleteSuggestionsPortal/TransparentOverlay/TransparentOverlay.tsx

```diff
@@ -29,14 +29,21 @@ function TransparentOverlay({onPress: onPressProp}: TransparentOverlayProps) {
         e?.preventDefault();
     }, []);
 
+    const overlay = useMemo(
+        () => ({
+            backgroundColor: 'rgba(0, 0, 0, 0.005)',
+        }),
+        [],
+    );
```

Comment:
I didn't want to put this in the styles constant because it feels a bit hacky. Plus, we usually avoid inline styling in the render method since it's not the best practice

Reply:
sure 👍🏻